gtp2ogs.js

```diff
@@ -305,6 +344,16 @@ class Game {
             this.state = gamedata;
             check_for_move();
         });
+        this.socket.on('game/' + game_id + '/clock', (clock) => {
```

Comment:
I've been working on this but I don't fully understand why the clock subscription isn't in here with the other Game-specific subscriptions like gamedata, phase, and move.